Sources/ComposableArchitecture/Store.swift

```diff
@@ -314,6 +314,30 @@ public final class Store<State, Action> {
     #endif
   }
 
+  /// Scopes the store to one that exposes child state and actions.
+  ///
+  /// A version of ``scope(state:action:)-9iai9`` that exposes the child state along
+  /// with the child action for scoping actions.
+  ///
+  /// - Parameters:
+  ///   - toChildState: A function that transforms `State` into `ChildState`.
+  ///   - fromChildAction: A function that transforms `ChildState` and `ChildAction` into `Action`.
+  /// - Returns: A new store with its domain (state and action) transformed.
+  @_disfavoredOverload
+  @_spi(Internals) public func scope<ChildState, ChildAction>(
```

Comment:
Added `@_spi` annotation as discussed on Slack. Hopefully this makes the addition not as disruptive to public API but still suitable for keen library developers.